### ISSUE TITLE
FGSP is not supported in Azure

### ISSUE THREAD

**Issue by jjeziorny** [fortinetsolutions/Azure-Templates#6]
FGSP uses Ethernet type 0x8892 which is not supported in Azure (or most public clouds for that matter). So the proposed configuration is not valid in Azure.

In short, it is NOT possible to synchronise sessions between Fortigate devices in Azure. This is quite disappointing and it would be great if Fortinet could release a fix for it. Until then, I suggest you remove the mentions of session synchronisation between devices as that's is misleading.

**Comment by fortinetclouddev** [comment]
Hi Juliano,

FGSP is capable of using 0x8892 ethertype for heartbeat.  However, TCP is also an option. 
See: http://help.fortinet.com/fos50hlp/54/Content/FortiOS/fortigate-ports-and-protocols-54/02-FGSP.htm

In most cases, including A/A HA using Azure's standard load balancers, Azure is not able to move active sessions in the case of a failover, so even though FGSP works, and sessions are synchronized, at probe failure, Azure will not move over current sessions (only new sessions get steered away from the failed appliances).   So, net is that having synchronized session states between the FortiGates doesn't help with failover.  

However, in the case mentioned in the readme - asymmetric routing - FGSP does work and is required to allow the solution to function.  Best practice would still be to avoid asymmetric routing, by leveraging Source-NAT for public connections and a single internal frontend for east/west connections.

**Comment by jjeziorny** [comment]
Hi,
Thanks for the reply, however it's still unclear to me how to configure FGSP to use TCP instead of ETHtype 0x8892.

The document mentions disabling session sync dev to allow it to use the hbdev for session synchronization, but it states it will use UDP for sync. 

Note that unsetting session-sync-dev (i.e. by entering unset session-syn-dev) has the following two effects:
    Session synchronization will use the ports defined as HA heartbeat interfaces (set hbdev).
    Session synchronization packets will be sent over UDP/708 instead of Ethertype 0x8892

During our tests we never saw sessions being synchronized, although it's been a while back we tested I'm pretty sure we tested without configuring a session sync device.

JJ
